### PR TITLE
Address Issue #35 - Contributing File

### PR DESCRIPTION
**Pull request by jk47**

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Directions
+
+To contribute to this repo, please fork and open a pull request.
+
+
+## Topics for Improvement
+* What is Ruby? (Week2)
+  * Elaborate on Dynamic Language
```
Comment (johnsonch) [inline contributing.md]: What would you suggest on this? Keeping in mind the course text book.

File: contributing.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+* What is REST? (Week3)
```
Comment (johnsonch) [inline contributing.md]: At this point of their class progression, I think the students may have a decent grasp. The new portions are `PUT` and `DELETE`. Typically they are coming with 2-3 semesters of Java or PHP.

File: contributing.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  * Talk about why REST pairs well with MVC
+  * Talk about how REST prevents the client from storing state
+    * Ensure there is an understanding of client/server
+* Objects (Week3)
+  * Might be useful to explain the hash rocket and what it means (=>)
```
Comment (johnsonch) [inline contributing.md]: Good point, I usually lecture more in class and expand on the topic but it ends up being a 'get off my lawn' rant about the older hash syntax.

File: contributing.md (hunk 1 above)
Comment (jk47) [reply contributing.md]: I just think its good to know the basics static versus dynamic --- variable declaration and initialization. I guess I don't know the knowledge /prereq's coming into the class.

File: contributing.md (hunk 2 above)
Comment (jk47) [reply contributing.md]: I see - wasn't sure on the prior classes/knowledge that the class would have.

File: contributing.md (hunk 3 above)
Comment (jk47) [reply contributing.md]: Haha - fair enough.

**Comment by jk47** [comment]
Adding a link to github's contributing guide that spells our the fork and PR process in detail.

**Pull request merged**